src/components/PeopleTableHead/PeopleTableHead.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { SearchLink } from '../SearchLink/SearchLink';
+import { OrderByType, SortBy } from '../../types/OrderAndSortTypes';
+
+const sortByValues = ['Name', 'Sex', 'Born', 'Died'];
+
+export const PeopleTableHead: FC = () => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const changeSortAndOrder = (sortByV: SortBy, orderByV: OrderByType) => {
+    const lowercaseSort = sortByV.toLowerCase();
```

Comment:
It is better not to shorten the names to unreadable ones.
Now it is not completely clear what exactly sortByV and orderByV mean.